modules/sttp-tapir/src/main/scala/io/janstenpickle/trace4cats/sttp/tapir/TapirResourceKleislis.scala

```diff
@@ -32,9 +33,10 @@ object TapirResourceKleislis {
     inHeadersGetter: Getter[I, Headers],
     inSpanNamer: TapirInputSpanNamer[I],
     errorToSpanStatus: TapirStatusMapping[E],
-    dropHeadersWhen: String => Boolean = HeaderNames.isSensitive
+    dropHeadersWhen: String => Boolean = HeaderNames.isSensitive,
+    errorHandler: PartialFunction[Throwable, SpanStatus] = PartialFunction.empty,
```

Comment:
Tapir injector has already got some sort of `errorToSpanStatus` mapping. I need to look at it carefully tomorrow.

Reply:
Yeah, good point. So does the http4s injector. I don't think we actually need this here, but child spans in a user's application can still benefit from it I think.